interfaz_controller/interfaz_controller.py

```diff
@@ -6,7 +6,8 @@ def main(workbook: xw.Book = None):
     wb = xw.Book.caller()
     sheet = wb.sheets[0]
 
-    params = sheet['B4:B8'].options(np.array, transpose=True).value
+    sheet.range('B4:B8').name = 'values'
```

Comment:
Not doing what is intented. When adding a row in excel the calculation throws an error.